src/auro-accordion.js

```diff
@@ -1,159 +1,103 @@
-// Copyright (c) 2020 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// Copyright (c) 2023 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
 // See LICENSE in the project root for license information.
 
 // ---------------------------------------------------------------------
 
-import { LitElement, html } from "lit-element";
-import { classMap } from 'lit-html/directives/class-map';
-
-// Import Icons
-import chevronUp from '@alaskaairux/icons/dist/icons/interface/chevron-up_es6.js';
-import chevronDown from '@alaskaairux/icons/dist/icons/interface/chevron-down_es6.js';
+// If using litElement base class
+import { LitElement, html } from "lit";
 
 // Import touch detection lib
-import "focus-visible/dist/focus-visible.min.js";
 import styleCss from "./style-css.js";
-import styleCssFixed from "./style-fixed-css.js";
 
 // See https://git.io/JJ6SJ for "How to document your components using JSDoc"
 /**
- * auro-accordion provides users a way to have collapsible content on a page.
- * Use auro-accordion-group if you want to have auto closing accordion components when others are selected.
- *
- * @element auro-accordion
+ * The auro-accordion element provides users a way to ... (it would be great if you fill this out).
```

Comment:
Not sure if this is coming from the same place as before (or the other way around) but we might want to update the copy here as well.